docs/terminology.md

```diff
@@ -0,0 +1,44 @@
+# Terminology
+Here we define some terms that are used in the project, with the goal of avoiding confusion and facilitating more
+accurate conversations related to `Authorino`.
+
+If you see terms used that are not here (or are used in place of terms here) please consider contributing a definition
+to this doc with a PR, or modifying the use elsewhere to align with these terms.
+
+## Terms
+* **Access token**: a type of temporary security token, tied to an authenticated identity, that is issued by an auth server under request of registered auth clients of this auth server, and that delegates power of operating on behalf of that identity to a party that presents this token on requests to a resource server, including and especially to the auth client itself; it is typically formatted as a JWT
```

Comment:
Maybe instead of...

```suggestion
* **Access token**: a type of temporary security credential, tied to an authenticated identity, that is issued by an auth server under request of registered auth clients of this auth server, and that delegates power of operating on behalf of that identity to a party that presents this token on requests to a resource server, including and especially to the auth client itself; it is typically formatted as a JWT
```

Reply:
Just to avoid defining it with the same word (token) and also because below we are saying that the AuthN process verifies credentials belonging to an identity.